extensions/BMO/template/en/default/bug/create/create-ipc.html.tmpl

```diff
@@ -0,0 +1,260 @@
+    <input type="hidden" name="component" value="Email">
+    <input type="hidden" name="rep_platform" value="All">
+    <input type="hidden" name="op_sys" value="Other">
+    <input type="hidden" name="version" value="unspecified">
+    <input type="hidden" name="priority" value="--">
+    <input type="hidden" name="bug_severity" value="normal">
+    <input type="hidden" name="format" value="ipc">
+    <input type="hidden" name="token" value="[% token FILTER html %]">
+    [% IF user.in_group('canconfirm') %]
+      <input type="hidden" name="bug_status" value="NEW">
+    [% END %]
+  
+    <div class="row">
+      <div class="row_desc">
+        <label class="required" for="team_name">
+          <strong>What team are you a part of?</strong>
+        </label>
+      </div>
+      <input required type="text" name="team_name" id="team_name" size="80">
+    </div>
+
+    <div class="row">
+      <div class="row_desc">
+        <label for="cf_user_story">
+          <strong>What is the user story associated with your snippet?</strong>
+        </label>
+      </div>
+      <textarea name="cf_user_story" id="cf_user_story" cols="80" rows="10"></textarea>
+    </div>
+
+    <div class="row">
+      <div class="row_desc">
+        <strong>Will your snippet need to appear in a language other than English? If so, please specify.</strong>
+      </div>
+      <input type="radio" name="language" id="spanish" value="Spanish">
+      <label for="spanish">Spanish</label><br>
+      <input type="radio" name="language" id="italian" value="Italian">
+      <label for="italian">Italian</label><br>
+      <input type="radio" name="language" id="german" value="German">
+      <label for="german">German</label><br>
+      <input type="radio" name="language" id="french" value="French">
+      <label for="french">French</label><br>
+      <input type="radio" name="language" id="russian" value="Russian">
+      <label for="russian">Russian</label><br>
+      <input type="radio" name="language" id="portuguese" value="Portuguese">
+      <label for="portuguese">Portuguese</label><br>
+      <input type="radio" name="language" id="langothersradio">
+      <label for="langothers">Others: </label>
+      <input type="text" name="language" id="langothers" size="30">
+    </div>
+
+    <div class="row">
+      <div class="row_desc">
+        <label for="target_country">
+          <strong>Does your snippet need to be targeted to a particular country? If so, please specify.</strong>
+        </label>
+      </div>
+      <input type="text" name="target_country" id="target_country" size="80">
+    </div>
+
+    <p>
+    If you would like your snippet to include a link, please tag it by using the URL builder <a href="http://bit.ly/1JOgDr1">(http://bit.ly/1JOgDr1)</a> and following the steps below:
+    </p>
+    <img src="https://bug1298932.bmoattachments.org/attachment.cgi?id=8786064" style = "width:50%"><br><br>
+    <div class="row">
```

Comment:
Anything for me to do on this? Should I download and keep it in the commit. But I unsure about how static files are served.